Project.toml

```diff
@@ -1,23 +1,24 @@
-authors = ["Andy Ferris <ferris.andy@gmail.com>"]
 name = "TypedTables"
 uuid = "9d95f2ec-7b3d-5a63-8d20-e2491e220bb9"
+authors = ["Andy Ferris <ferris.andy@gmail.com>"]
 version = "1.4.0"
 
 [deps]
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 Dictionaries = "85a47980-9c8c-11e8-2b9f-f7ca1fa99fb4"
 Indexing = "313cdc1a-70c2-5d6a-ae34-0150d3930a38"
+RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
 SplitApplyCombine = "03a91e81-4c3e-53e1-a0a4-9c0c8f19dd66"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
 
 [compat]
-julia = "1"
 Adapt = "1, 2, 3"
+Dictionaries = "0.3"
 Indexing = "1"
 SplitApplyCombine = "1"
 Tables = "1"
-Dictionaries = "0.3"
+julia = "1"
```

Comment:
Will need to add a compat entry here for RecipesBase or the General registry will reject a new release